src/App.js

```diff
@@ -1,24 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Nav from './components/Nav/Nav';
+import Weather from './pages/Weather/Weather';
+import WeatherDetails from './pages/Weather/WeatherDetails';
 
 function App() {
```

Comment:
- [ ] Your app should follow the general design guidelines including Typography (font face, size, and weight), Layout, and spacing, kindly make sure your design follows these requirements

|Your App|Behace design|
|-|-|
|![Screenshot_20220625_221310](https://user-images.githubusercontent.com/25227022/175790762-3d2c6853-350f-4100-b0f1-d1d73f313a44.png)|![Screenshot_20220625_221736](https://user-images.githubusercontent.com/25227022/175790763-62f5af42-b26b-4f43-be61-9ad63298b58c.png)|
|![Screenshot_20220625_221327](https://user-images.githubusercontent.com/25227022/175790776-0241587b-69f3-4c64-9d37-b2e5fdc0ee78.png)|![Screenshot_20220625_221757](https://user-images.githubusercontent.com/25227022/175790767-575e0944-5eb8-4a48-9aea-8f5a8dba49af.png)|